tests/kuttl/test-lifecycle-hook/01-pods.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdEnvironment
+metadata:
+  name: test-lifecycle-hook
+spec:
+  targetNamespace: test-lifecycle-hook
+  providers:
+    web:
+      port: 8000
+      mode: operator
+    metrics:
+      port: 9000
+      mode: operator
+      path: "/metrics"
+    kafka:
+      mode: none
+    db:
+      mode: none
+    logging:
+      mode: none
+    objectStore:
+      mode: none
+    inMemoryDb:
+      mode: none
+  resourceDefaults:
+    limits:
+      cpu: 400m
+      memory: 1024Mi
+    requests:
+      cpu: 30m
+      memory: 512Mi
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdApp
+metadata:
+  name: lifecycle
+  namespace: test-lifecycle-hook
+spec:
+  envName: test-lifecycle-hook
+  deployments:
+  - name: processor
+    podSpec:
+      image: quay.io/psav/clowder-hello
+      lifecycle:
+        # postStart:
```

Comment:
If this is commented-out can we remove it?

Reply:
I really left this here to show how else it could be used, but I'll remove it since there is an example in this PR description.